framework/include/utils/RankTwoTensorImplementation.h

```diff
@@ -1125,26 +1141,19 @@ template <typename T>
 RankTwoTensorTempl<T>
 RankTwoTensorTempl<T>::genRandomTensor(T scale, T offset)
```

Comment:
Could we rename it to simply `rand`, the words "gen" and "Tensor" are not describing anything useful, especially when this is used as a named constructor, e.g. `RankTwoTensor::genRandomTensor(1, 0)` vs `RankTwoTensor::random(1, 0)`.

Reply:
The new version that scales first and then adds the offset will be names `rand` and `symmetricRand`

Noted in #21269